releasenotes/notes/fix-dbapi-async-aexit-0df48d739dfd2a3a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    dbapi-async: Pass `exc_type`, `exc_val`, `exc_tb` down to the wrapped object on `__aexit__` calls.
```

Comment:
Currently this bug only impacts the psycopg integration.
```suggestion
    psycopg: Resolves ``TypeError`` raised when an async cursor object is traced. This fix ensures `exc_type`, `exc_val`, and `exc_tb` are passed down to the wrapped object on `__aexit__`.
```

Reply:
Should note, the change affected the dbapi_async implementation, which is why I scoped the fix with that category, any async dbapi client could be affected by this, theoretically.